### ISSUE TITLE
Fix translatable strings according to GlotPress

### ISSUE THREAD

**Issue by DevinWalker** [WordImpress/Give#451]
Now that we are added on GlotPress I'm seeing issues with the strings that are being translated. We should be providing descriptions using `_x()` https://codex.wordpress.org/Function_Reference/_x more often and reduing the amount of HTML within the strings found here: https://translate.wordpress.org/projects/wp-plugins/give

**Comment by DevinWalker** [comment]
Closing this due to it being a vague issue and all the recent PRs by @ramiy https://github.com/WordImpress/Give/pull/710 and https://github.com/WordImpress/Give/pull/707
